src/main/java/gov/cabinetoffice/gap/adminbackend/services/ApplicationFormService.java

```diff
@@ -232,6 +233,14 @@ public String addQuestionToApplicationForm(Integer applicationId, String section
         String questionId = UUID.randomUUID().toString();
 
         this.applicationFormRepository.findById(applicationId).ifPresentOrElse(applicationForm -> {
+            ApplicationFormSectionDTO sectionDTO = null;
```

Comment:
Don't need to initialise this with `null`

`ApplicationFormSectionDTO sectionDTO;`